model_monitoring/components/feature_attribution_drift/feature_attribution_drift_compute_metrics/spec.yaml

```diff
@@ -46,8 +53,3 @@ conf:
         - responsibleai~=0.27.0
         - scikit-learn<1.1
   name: responsibleai-spark
```

Comment:
can you ensure the conf is not the last thing in the file? any presence of a bad space (aka new line) will result in Spark creating a new synapse cluster altogether.

Reply:
ahh i was wondering why the order of the yaml got changed; updated!